input/resources/au-core-requester.xml

```diff
@@ -1504,6 +1505,7 @@
 <div>
 <p>If the requester supports the MedicationRequest resource, the requester <strong>SHALL</strong> support the AU Core profile and the conformance expectations for the MedicationRequest resource.</p>
 <p>The MedicationRequest resources can represent a medication using either a code or refer to the Medication resource. When referencing Medication, the resource may be <a href="http://hl7.org/fhir/R4/references.html#contained">contained</a> or an external resource. The responder <strong>MAY</strong> choose any one way or more than one method, but if an external reference to Medication is used, the responder <strong>SHALL</strong> support the <code>_include</code> parameter for searching this element. The requester application <strong>SHALL</strong> support all methods.</p>
+<p>The MedicationRequest resource can communicate the clinical indication using either a code in MedicationRequest.reasonCode or a reference using MedicationRequest.reasonReference. Although both <code>MedicationRequest.reasonCode</code> and <code>MedicationRequest.reasonReference</code> are marked as Must Support, a responder is not required to support both but <strong>SHALL</strong> support at least one of these elements. The requester <strong>SHALL</strong> support both elements.</p>
```

Comment:
Apply formatting to element names consistently.

Reply:
The absence of formatting in the beginning was intentional in all of these instances. But it's fine to change it.